CDP4Composition/Reporting/ReportingDataSourceColumn.cs

```diff
@@ -0,0 +1,69 @@
+﻿// --------------------------------------------------------------------------------------------------------------------
+// <copyright file="ReportingDataSourceParameter.cs" company="RHEA System S.A.">
```

Comment:
is this the best place to put this class, why here and not in the reporting plugin? do we expect to reuse this in other plugins?

Reply:
It would indeed be better to move all Reporting classes from Composition to the Reporting plugin. They were added here because this is how the initial demo implementation was started on the `feat/report-branch` branch.